exchanges/gateio/gateio.go

```diff
@@ -1345,6 +1345,29 @@ func (g *Gateio) GetUsersTotalBalance(ctx context.Context, ccy currency.Code) (*
 	return response, g.SendAuthenticatedHTTPRequest(ctx, exchange.RestSpot, walletTotalBalanceEPL, http.MethodGet, walletTotalBalance, params, nil, &response)
 }
 
+// ConvertSmallBalances converts small balances of supported crypto (excluding delisted crypto) above 0.00000001 GT and
+// below 0.0001 BTC in value to GT. You can make a conversion every 6 hours. A quota of 100 GT is available per account
+// every 3 days.
+// Providing a list of currency codes will convert only those, else all small currencies will be converted.
+func (g *Gateio) ConvertSmallBalances(ctx context.Context, codes ...currency.Code) error {
+	currencyList := make([]string, len(codes))
+	for i := range codes {
+		if codes[i].IsEmpty() {
+			return currency.ErrCurrencyCodeEmpty
+		}
+		currencyList[i] = codes[i].Upper().String()
+	}
+
+	body := struct {
```

Comment:
🐒 Is there a reason this isn't a `smallCurrencyReq` type?
🥃 `body` is completely fine. `payload` is slightly better IMO.
```suggestion
	payload := struct {
```